third_party/amd/lib/TritonAMDGPUTransforms/StreamPipelineV2.cpp

```diff
@@ -295,16 +295,14 @@ void StreamPipeliner::assignMemoryLayouts() {
 
     // Limit shared memory sharing to width >= 32 elements.
     LDBG("Load " << *loadOp << " has width " << width);
-    if (width < 32) {
```

Comment:
StreamPipelineV2.cpp change in this PR enables pipelining in registers. This change was suggested to me by Simon.

Reply:
Yes, it could be posted separately. @oplavsic there should be other cases that will exercise your pass right?

Reply:
This is now in https://github.com/triton-lang/triton/pull/5227. Let's drop it here.